WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -196,6 +196,10 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
         uploadsManager.resume()
         updateFeatureFlags()
         updateRemoteConfig()
+        if let windowManager = windowManager as? JetpackWindowManager,
```

Comment:
It will try to import here as long as the migration completion flag is false. BUT, the import will fail so nothing will happen if the user hasn't triggered and export. And after migration flow is done once on JP, the if will return false so it will never be executed again.

Reply:
Sounds good @alpavanoglu. But I just want to point out that WordPress target doesn't compile because `JetpackWindowManager` type is Jetpack-only.